web/src/beta/features/ProjectSettings/innerPages/StorySettings/StorySettingsDetail.tsx

```diff
@@ -16,18 +18,33 @@ type Props = {
 const StorySettingsDetail: React.FC<Props> = ({ settingsItem, onUpdateStory }) => {
   const t = useT();
 
-  // TODO: select filed
-  const [localPanelPosition] = useState(settingsItem.panelPosition);
+  const [localPanelPosition, setLocalPanelPosition] = useState<Position | undefined>(
+    settingsItem.panelPosition,
+  );
   const handleSubmit = useCallback(() => {
     onUpdateStory({
       panelPosition: localPanelPosition,
     });
   }, [localPanelPosition, onUpdateStory]);
 
+  const options = useMemo(
+    () =>
+      Object.keys(Position).map(label => ({
+        label,
+        key: Position[label as keyof typeof Position],
+      })),
+    [],
+  );
```

Comment:
I don't think this will support internationalization. Can you update this code to include it please 🙏🏻

Reply:
So when i tried 
```
label: t(label) 
```
i18n didn't pick up the translation probably because it wasn't string so I just used a variable rather than this loop.